test/remote/gateways/remote_ixopay_test.rb

```diff
@@ -53,43 +53,45 @@ def test_failed_authentication
     assert_equal '1004', response.error_code
   end
 
-  def test_successful_authorize#_and_capture
+  def test_successful_authorize_and_capture
     auth = @gateway.authorize(@amount, @credit_card, @options)
+
     assert_success auth
     assert_equal 'FINISHED', auth.message
     assert_not_nil auth.params['purchase_id']
     assert_not_nil auth.params['reference_id']
     assert_not_nil auth.authorization
 
-    #assert capture = @gateway.capture(@amount, auth.authorization)
-    #assert_success capture
-    #assert_equal 'REPLACE WITH SUCCESS MESSAGE', capture.message
-  end
-
-  def test_failed_authorize
-    response = @gateway.authorize(@amount, @declined_card, @options)
-    assert_failure response
-    assert_equal 'The transaction was declined', response.message
-    assert_equal 'ERROR',                        response.params['return_type']
-    assert_equal response.error_code, "2003"
+    assert capture = @gateway.capture(@amount, auth.params['reference_id'])
+    assert_success capture
+    assert_equal 'FINISHED', capture.message
   end
 
   def test_partial_capture
-    omit 'Not yet implemented'
+    auth_amount = 500
+    capture_amount = 400
```

Comment:
What is the advantage of adding these additional lines of code instead of following the default of using `@amount-1`?

Reply:
I wrote this before I noticed that there was already a ```test_partial_capture```, so I wasn't going for a particular advantage.

Reply:
(Also, changed it to the subtraction version)